server/shared/mail/templates/components/header.mjml

```diff
@@ -1,23 +1,15 @@
-<mj-section padding="10px" text-align="left">
-  <mj-group width="130px">
-    <mj-column
-      width="18%"
-      background-color="#E91D63"
-      border-radius="4px">
-      <mj-text
-        padding="4px 15px"
-        color="#fff"
-        font-size="16px"
-        align="center">
-        T
-      </mj-text>
-    </mj-column>
-    <mj-column width="82%">
-      <mj-text
-        mj-class="title"
-        padding="3px 24px">
-        Tailor
-      </mj-text>
-    </mj-column>
-  </mj-group>
+<mj-section>
+  <mj-column>
+    <mj-table>
```

Comment:
Why table for logo?

Reply:
To achieve border radius 50%. Since on mac OS mail client it is still not perfectly round, I think that new svg image will be needed after.